src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -210,3 +167,124 @@ const getStyles = cacheStyles((theme: Theme) => ({
     minHeight: theme.rem(6)
   }
 }))
+
+const setTouchOtp = async (account: EdgeAccount, dispatch: Dispatch) => {
+  await enableTouchId(account).catch(e => {
+    console.log(e) // Fail quietly
+  })
+  await account.dataStore.setItem(
+    Constants.OTP_REMINDER_STORE_NAME,
+    Constants.OTP_REMINDER_KEY_NAME_CREATED_AT,
+    Date.now().toString()
+  )
+  dispatch(loadTouchState())
+}
+
+/**
+ * Terms of Service scene for new regular or light accounts
+ */
+interface NewAccountTosProps extends SceneProps<'newAccountTos'> {
+  branding: Branding
+}
+export const NewAccountTosScene = (props: NewAccountTosProps) => {
+  const { route, branding } = props
+  const imports = useImports()
```

Comment:
Change this line do `const {context, accountOptions} = useImports`, and then stop using `imports` later in the function.